src/Exception/CitizenProjectCommitteeSupportNotFoundException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AppBundle\Exception;
+
+use AppBundle\Entity\Committee;
+use Throwable;
+
+class CitizenProjectCommitteeSupportNotFoundException extends CitizenProjectCommitteeSupportException
```

Comment:
Not sure this exception has a true value, what about throwing an `\InvalidArgumentException` or a `\RuntimeException` from the manager?

Reply:
I create this exception to precise exception type and catch it more simply, I use this exception in `deleteCommitteeSupport` on Manager